src/node_api.cc

```diff
@@ -532,10 +543,11 @@ class CallbackWrapperBase : public CallbackWrapper {
       : CallbackWrapper(JsValueFromV8LocalValue(cbinfo.This()),
                         args_length,
                         nullptr),
-        _cbinfo(cbinfo),
-        _cbdata(v8::Local<v8::Object>::Cast(cbinfo.Data())) {
-    _data = v8::Local<v8::External>::Cast(_cbdata->GetInternalField(kDataIndex))
-                ->Value();
+        _cbinfo(cbinfo) {
+    // Note that there is no way we can tell whether `_bundle` is legit
```

Comment:
nit: AFAICT the fact that V8 has chosen to call **this** function rather than some other function proves that `_bundle` is legit, because the only way that can happen is if we associated `_bundle` with this JavaScript function during module initialization. So, I don't believe this comment is necessary.